### DISCUSSION TITLE
Mockingbird themes

### DISCUSSION THREAD

**Discussion by Felipe-gsilva** [moclojer/mockingbird#5] [Ideas]
@avelino @matheusfrancisco @J0sueTM 

Team, I've been thinking about some default themes to add on mockingbird and I've came up with some ideas. When I say themes I mean repainting LOL, just changing the colors.

The idea was to add 3 themes, inspired, just as our design system name, night cycle birds, like:
- **mockingbird:** the default experience we use on our projects, defined by the use of a strong pink and some contrast on it.

- **nightjar:** yeah, there is a bird called NIGHT JAR, which seems like a nature signal for us to add it here. I tought it should be based on the actual bird, mixing some dark gray and some low saturated yellow (#36362C and #E0D94A)
![image](https://github.com/user-attachments/assets/e55bf734-3b8c-4bc7-8ca0-58fa235a2efc)![image](https://github.com/user-attachments/assets/dce978fb-47ed-4874-94fb-bd28fc3bc57c)

- **kiwi:** this would be a theme based on yet another bird with a odd name and should include a variety of also dark gray, but contrasted with light green (matching also the fruit)
![image](https://github.com/user-attachments/assets/075c54d1-4952-4c8b-ad07-799c500023b2)![image](https://github.com/user-attachments/assets/561040cc-fd8b-4a76-9151-02e8288600ed)

what do you guys think?

**Comment by avelino** [comment]
I like the idea of having 3 standard themes, mainly because of the "name play" with birds.

I liked the proposal of the colors.

**Comment by J0sueTM** [comment]
I really like the idea. Give the enduser customizability, but not that much (for now). Btw, this won''t be mapped in our current sprint, right?
